src/utils/datetime.js

```diff
@@ -110,3 +110,18 @@ export function millisecondsAsTimeOfDay(milliseconds, format = 'h:mm a') {
   }
   return moment.utc(milliseconds).format(format);
 }
+
+/**
+ * formatDisplayDate
+ * @param  {Number} timestamp integer hammertime
+ * @param  {Object} timePrefs object containing timezoneAware Boolean and timezoneName String
+ *
+ * @return {String}           formatted timezoneAware date string
+ */
+export function formatDisplayDate(timestamp, timePrefs) {
+  if (!timestamp) {
+    return '';
+  }
+  return moment(timestamp).utc().tz(getTimezoneFromTimePrefs(timePrefs))
```

Comment:
It's really strange. I put together a set of console logs to try and figure out what it was doing, but it's clear that the `moment().utc(<arg>)` version is significantly different from the `moment(<arg>).utc()` version. Maybe you can make better sense of what's happening, but it seems to be somehow relative to the current time if we do `moment().utc(<arg>)` since the two `.valueOf()` calls are one millisecond apart, inferring some execution time passed between them.
```
time (string):                 2016-09-13T16:59:52.000Z
hrmt - Date.parse(time):       1473785992000
moment().utc(time).valueOf():  1473880342561
moment().utc(hrmt).valueOf():  1473880342562
moment(time).utc().valueOf():  1473785992000
moment(hmrt).utc().valueOf():  1473785992000
moment(time).format():         2016-09-13T12:59:52-04:00
moment(hmrt).format():         2016-09-13T12:59:52-04:00
moment(time).utc().format():   2016-09-13T16:59:52Z
moment(hmrt).utc().format():   2016-09-13T16:59:52Z
moment().utc(time).format():   2016-09-14T19:12:22Z
moment().utc(hmrt).format():   2016-09-14T19:12:22Z
moment().utc().format():       2016-09-14T23:12:22Z
```